documentation/identifiers.md

```diff
@@ -1,70 +1,80 @@
 # Identifiers
 
-## Why identifiers matter
+```eval_rst 
 
-Identifiers are an important part of any dataset. They let a computer uniquely identify and refer to specific grants, organisations, transactions and so-on. 
+.. admonition:: Why identifiers matter
 
-Whilst a human being may be good at recognising that:
+  Identifiers are an important part of any dataset. They let a computer uniquely identify and refer to specific grants, organisations, transactions and so-on. 
 
->INDIGO TRUST, The Indigo Trust, and indigo-trust
+  Whilst a human being may be good at recognising that "INDIGO TRUST", "The Indigo Trust", and "indigo-trust" all refer to the same organisation, computers find this a lot trickier. 
 
-... all refer to the same organisation, computers find this a lot trickier. 
+```
 
-That's why 360Giving requires you to give identifiers to:
+360Giving asks you to give identifiers to:
 
-* Grants;
+* [Grants](grant-identifier)
+* [Organisations](organisation-identifier);
 * Transactions; 
 * Classifications;
```

Comment:
Can we give an example from the schema like the above? Also, I am not sure people are using the transactions and classifications and if it's not going to confuse them even more.

Reply:
To keep things concise I would suggest not adding too many examples here, but I'll update sentence before to make clear people may not always have all of these:

"360Giving asks you to give identifiers to any of the following elements that you include in your data:"